modules/mongodb/manifests/monitoring.pp

```diff
@@ -1,4 +1,7 @@
-# FIXME: This class needs better documentation as per https://docs.puppetlabs.com/guides/style_guide.html#puppet-doc
+# == Class: mongodb::monitoring
+#
+# Configures monitoring for mongodb
+#
 class mongodb::monitoring ($dbpath = '/var/lib/mongodb') {
```

Comment:
This should be reformatted to a new line and have a `Parameter` section explaining it.